src/centered-on-asteroid/train.py

```diff
@@ -141,27 +148,79 @@ def train(
     return best_model
 
 
+def get_experiment_args():
+    parser = ArgumentParser()
+    parser.add_argument("--image_shape", type=int, nargs=2, default=(30, 30))
+    parser.add_argument("--images_per_sequence", type=int, default=4)
+    parser.add_argument("--feature_vector_size", type=int, default=10)
+    parser.add_argument("--loss", type=str, default="BCELoss")
+    parser.add_argument("--optimizer", type=str, default="Adam")
+    parser.add_argument("--epochs", type=int, default=10)
+    parser.add_argument("--batch_size", type=int, default=4)
+    parser.add_argument("--experiment_name", type=str, default="end-to-end")
+    parser.add_argument("--metadata", type=str, default="None")
+    return parser.parse_args()
+
+
 if __name__ == "__main__":
+    # Loss lookup table
+    classification_loss_functions = {
+        "CrossEntropyLoss": nn.CrossEntropyLoss(),
+        "NLLLoss": nn.NLLLoss(),
+        "BCELoss": nn.BCELoss(),
+        "BCEWithLogitsLoss": nn.BCEWithLogitsLoss(),
+        "MultiLabelSoftMarginLoss": nn.MultiLabelSoftMarginLoss(),
+        "MultiMarginLoss": nn.MultiMarginLoss(),
+        "SoftMarginLoss": nn.SoftMarginLoss(),
+    }
+    args = get_experiment_args()
     # Model parameters
-    image_shape = (30, 30)
-    images_per_sequence = 4
-    feature_vector_size = 10
+    image_shape = args.image_shape
+    images_per_sequence = args.images_per_sequence
+    feature_vector_size = args.feature_vector_size
 
-    model = model_def.CFN(images_per_sequence, feature_vector_size, image_shape)
+    # Load data
+    # Ignore weird paths for now
+    path_to_data = os.path.abspath("./../../scripts/data")
+    real_movers_file = f"{path_to_data}/csv/csv/movers_cond_12_image_meta_data.csv"
+    bogus_movers_file = f"{path_to_data}/csv/csv/movers_cond_2_image_meta_data.csv"
```

Comment:
I guess ideally these would be parameters too. Long term I don't want to have two separate files, instead just one that contains the labels, means we can curate a larger dataset.